emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBConstants.java

```diff
@@ -27,8 +27,9 @@ public interface DynamoDBConstants {
   String DYNAMODB_ACCESS_KEY_CONF = "dynamodb.awsAccessKeyId";
   String DYNAMODB_SECRET_KEY_CONF = "dynamodb.awsSecretAccessKey";
   String DYNAMODB_SESSION_TOKEN_CONF = "dynamodb.awsSessionToken";
-  String DEFAULT_ACCESS_KEY_CONF = "fs.s3.awsAccessKeyId";
-  String DEFAULT_SECRET_KEY_CONF = "fs.s3.awsSecretAccessKey";
+  String DEFAULT_ACCESS_KEY_CONF = "fs.s3a.access.key";
+  String DEFAULT_SECRET_KEY_CONF = "fs.s3a.secret.key";
+  String DEFAULT_SESSION_TOKEN_CONF = "fs.s3a.session.token";
```

Comment:
Note that I also changed the `DEFAULT_ACCESS_KEY_CONF` and `DEFAULT_SECRET_KEY_CONF` to be consistent with the session token property (see [here](https://github.com/apache/spark/blame/v3.0.0/core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala)).